roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -1,18 +1,44 @@
 ---
 - name: collect admin and bootstrap keys
+  command: ceph-create-keys --cluster {{ cluster }} -i {{ monitor_name }} -t 30
+  args:
+    creates: /etc/ceph/{{ cluster }}.client.admin.keyring
+  failed_when: false
```

Comment:
Could you please remove this? Otherwise we won't fail if mons are not in quorum.
Thanks!

And also please rebase on master first.